ansible/roles/texlive/tasks/main.yml

```diff
@@ -0,0 +1,63 @@
+- name: texlive | prereq
+  yum: name={{ item }} state=latest
+  with_items:
+    - wget
+    - perl-Tk
+    - perl-Digest-MD5
+
+- name: texlive | download and extract installer
+  unarchive:
+    copy: no
+    creates: /tmp/install-tl-unx.tar.gz
+    dest: /tmp/
+    src: http://mirror.unl.edu/ctan/systems/texlive/tlnet/install-tl-unx.tar.gz
+
+- name: texlive | copy profile
+  copy: src=install-tl.profile dest=/tmp/install-tl.profile
```

Comment:
Template done. It's much nicer. I had started looking at using a template, but for some reason was put off - it seemed like it could be more complex than it was.